website/datafuse/docs/api/config.md

```diff
@@ -8,7 +8,7 @@ Get the config of the Datafuse query server.
 ## Examples
 
 ```
-curl http://127.0.0.1:8080/v1/configs
+curl http://127.0.0.1:8081/v1/configs
```

Comment:
There are many relevant ports that are 8080, so we should make changes in a separate PR and make sure that all descriptions are for the same specific run